### PR TITLE
[15.0][ADD] account_chart_update_l10n_eu_oss

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Glue module for account chart update and OSS. It makes the wizard ignore OSS
taxes when updating the CoA.

@ForgeFlow

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Nitpicking: as it's an extension for account_chart_update, better to name it `account_chart_update_l10n_eu_oss`

**Comment by LoisRForgeFlow** [comment]
> Nitpicking: as it's an extension for account_chart_update, better to name it `account_chart_update_l10n_eu_oss`

I was doubtful with the name, what you propose makes probably more sense, it is already changed.

File: account_chart_update_l10n_eu_oss/wizard/wizard_chart_update.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+from odoo import models
+
+
+class WizardUpdateChartsAccounts(models.TransientModel):
+    _inherit = "wizard.update.charts.accounts"
+
+    def _domain_taxes_to_deactivate(self, found_taxes_ids):
+        domain = super()._domain_taxes_to_deactivate(found_taxes_ids)
+        domain.append(("tax_group_id.name", "not like", "OSS%%"))
```
Comment (pedrobaeza) [inline account_chart_update_l10n_eu_oss/wizard/wizard_chart_update.py:12]: Better to use the specific field `oss_country_id`

Comment (LoisRForgeFlow) [reply account_chart_update_l10n_eu_oss/wizard/wizard_chart_update.py:12]: I cannot find where that field is defined, could you point it to me?

Comment (pedrobaeza) [reply account_chart_update_l10n_eu_oss/wizard/wizard_chart_update.py:12]: https://github.com/OCA/account-fiscal-rule/blob/14.0/l10n_eu_oss/models/account_tax.py#L10

But this is for the OCA module. Maybe you are not using it?

Comment (pedrobaeza) [reply account_chart_update_l10n_eu_oss/wizard/wizard_chart_update.py:12]: The 15.0 reference: https://github.com/OCA/account-fiscal-rule/blob/15.0/l10n_eu_oss_oca/models/account_tax.py#L10

Comment (LoisRForgeFlow) [reply account_chart_update_l10n_eu_oss/wizard/wizard_chart_update.py:12]: Hmmm I didn't know that module, I'm using the standard one https://github.com/odoo/odoo/tree/15.0/addons/l10n_eu_oss. Do you have some insight on why the OCA alternative was needed? In other words, what is the problems with the standard one (l10n_eu_oss)?

Comment (pedrobaeza) [reply account_chart_update_l10n_eu_oss/wizard/wizard_chart_update.py:12]: It doesn't cover correctly B2C/B2B flows at the same time, as the auto-application of the fiscal positions can't cover correctly all the cases.

Comment (pedrobaeza) [reply account_chart_update_l10n_eu_oss/wizard/wizard_chart_update.py:12]: And ours was first.

Comment (LoisRForgeFlow) [reply account_chart_update_l10n_eu_oss/wizard/wizard_chart_update.py:12]: I see, I think we need to cover both cases for those who have the Odoo one already installed. I can create another glue module `account_chart_update_l10n_eu_oss_oca` for the OCA alternative, WDYT?

Comment (pedrobaeza) [reply account_chart_update_l10n_eu_oss/wizard/wizard_chart_update.py:12]: OCA, but Odoo link is very weak and error prone. I urge you to switch to OCA one.

Comment (LoisRForgeFlow) [reply account_chart_update_l10n_eu_oss/wizard/wizard_chart_update.py:12]: What I meant above is we can have both `account_chart_update_l10n_eu_oss_oca` and `account_chart_update_l10n_eu_oss`, is this feasible for you?

> I urge you to switch to OCA one.

I'll try!

Comment (pedrobaeza) [reply account_chart_update_l10n_eu_oss/wizard/wizard_chart_update.py:12]: Yes, of course.

**Review by pedrobaeza — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Code review LGTM

**Comment by pedrobaeza** [comment]
The copier template should be updated.

**Comment by pedrobaeza** [comment]
WTF?

**Pull request merged**